sonar-docs/analysis/scan/sonarscanner-for-msbuild.md

```diff
@@ -36,17 +33,17 @@ It supports .Net Core multi-platform projects and it can be used on non-Windows
 * Java 11 or greater
 <!-- /sonarcloud -->
 * The SDK corresponding to your build system:
-   *  [.NET Framework v4.6](https://www.microsoft.com/fr-fr/download/details.aspx?id=53344) - either [Build Tools for Visual Studio 2015 Update 3](https://go.microsoft.com/fwlink/?LinkId=615458) or the [Build Tools for Visual Studio 2017](https://www.visualstudio.com/downloads/)
-   * [.NET Core SDK 2.0 and above](https://www.microsoft.com/net/learn/get-started/) (for .NET Core version of the scanner or if you plan to use [.NET Core Global Tool](https://www.nuget.org/packages/dotnet-sonarscanner)
+   * [.NET Framework v4.6](https://www.microsoft.com/fr-fr/download/details.aspx?id=53344) - either [Build Tools for Visual Studio 2015 Update 3](https://go.microsoft.com/fwlink/?LinkId=615458) or the [Build Tools for Visual Studio 2017](https://www.visualstudio.com/downloads/)
+   * [.NET Core SDK 2.0 and above](https://dotnet.microsoft.com/download) (for .NET Core version of the scanner or if you plan to use [.NET Core Global Tool](https://www.nuget.org/packages/dotnet-sonarscanner)
 
 [[info]]
-| The flavor used to compile the Scanner for .NET (either .NET Framework or .NET Core) is independant of the .NET version the 
-| project you want to analyze has been built with. Concretely, you can analyze .NET Core code with the .NET Framework version of 
+| The flavor used to compile the Scanner for .NET (either .NET Framework, .NET Core or .NET) is independant of the .NET version the 
```

Comment:
> .NET Framework, .NET Core or .NET
what is ".NET" in this case?

Reply:
https://docs.microsoft.com/fr-fr/dotnet/core/dotnet-five :D